src/components/Strategies/StrategyList.tsx

```diff
@@ -1,59 +1,45 @@
+import { strategies } from './strategyData';
+
 const { Meta } = Card;
 
 const { Text } = Typography;
 
 function StrategyList() {
+  const strategyClick = (strategy: IStrategies) => {
+    try {
+      (window as any).heap.track('StrategyClicked', { strategy: strategy.title });
+    } catch (e) {
+      console.error(e as any);
+    }
+    const comingSoonTextId = document.getElementById(strategy.title);
+    if (strategy.displayComingSoonText) {
+      comingSoonTextId!.classList.remove('invisible');
+    } else {
+      window.open('/' + strategy.detailPageURL);
+    }
+  };
+
   return (
     <Container>
       <Row gutter={[24, { xs: 8, sm: 16, md: 24, lg: 32 }]}>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/atlas-mine.jpg" />}>
-            <Meta title="Claim Rewards" description="Bridgeworld (Atlas Mine)" />
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/magic-dragon.jpg" />}>
-            <Meta title="Claim Rewards" description="Magic Dragon DAO" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/battlefly.jpg" />}>
-            <Meta title="Claim Rewards" description="BattleFly" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/tales-of-alleria.jpg" />}>
-            <Meta title="Claim Rewards" description="Tales of Elleria" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card hoverable cover={<img alt="example" src="../img/smithy.jpg" />}>
-            <Meta title="Go on Quests" description="SmithyDAO" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
-        <Col span={8}>
-          <Card className="antcard" hoverable cover={<img alt="example" src="../img/ivory-tower.jpg" />}>
-            <Meta title="Send Legions Questing" description="Bridgeworld" />
-            <Text type="danger" className="soon">
-              COMING SOON!
-            </Text>
-          </Card>
-        </Col>
+        {strategies.map((strategy) => (
+          <Col span={8} key={strategy.itemId}>
+            <div onClick={() => strategyClick(strategy)}>
+              <Card hoverable cover={<img alt={strategy.title} src={strategy.imageSrc} />}>
+                <Meta title="Claim Rewards" description={strategy.title} />
+                <Text type="danger" className="comingSoonText">
+                  <div className="invisible" id={strategy.title}>
```

Comment:
Why the nested div?

Also, since you have the `id` in `strategy` why not use that? It's guaranteed to be unique